apps/industrial-hemp/views/content/en/thc-application-submitted.md

```diff
@@ -0,0 +1,7 @@
+Application reference number: 12345678
+
+An email confirming your application and a PDF of your details has been sent to {{values.licence-email-address}}.
```

Comment:
Might be worth both us confirming with ucd on how the email should look, for registration it is a gov-uk link but here it seems it's just plain text